modules/avoindata-drupal-header/templates/avoindata_header_search_collapsible.html.twig

```diff
@@ -6,13 +6,20 @@
  * @ingroup themeable
  */
 #}
-<div class="search-collapsible">
-    <div class="collapse width" id="search-collapse">
-      <input id="avoindata-nav-search-input" class="form-control" placeholder="Etsi tietoaineistosta..." value="">
-    </div>
-    <div class="search-collapsible-control">
-      <a href="#search-collapse" data-toggle="collapse">
-        <span class="fas fa-search header-icon" aria-hidden="true">
-      </a>
-    </div>
+{{ attach_library('avoindata_header/avoindata_header') }}
+<div id="header-search-container" hidden>
+  <div class="header-search-input-container is-hidden">
+    {% set ckanLang = language %}
+    {% if ckanLang == 'en' %}
+      {% set ckanLang = 'en_GB' %}
+    {% endif %}
+    <form action="/data/{{ ckanLang }}/dataset" id="header-search-form" method="get">
+      <input name="q" class="form-control" id="avoindata-nav-search-input" placeholder="{% trans %}Etsi tietoaineistosta...{% endtrans %}" value="">
```

Comment:
"Etsi tietoaineistoista" probably should not be in finnish even though it was that before too.

```suggestion
      <input name="q" class="form-control" id="avoindata-nav-search-input" placeholder="{% trans %}Search datasets...{% endtrans %}" value="">
```